docs/releasing.md

```diff
@@ -1,27 +1,17 @@
 # Releasing
 
-## Test Locally
-
-`go test ./...`
-
-## Push new docs
+## Release to production
 
-build docs locally: `make site`
+This can all be done from your local terminal.
 
-build and push docs to production: `make site-docs`
+1. `git tag v1.2.3`
+2. `git push origin v1.2.3`
+3. Wait a few minutes for the build to run <https://github.com/cli/cli/actions>
+4. Check <https://github.com/cli/cli/releases>
 
 ## Release locally for debugging
 
 A local release can be created for testing without creating anything official on the release page.
 
-1. `env GH_OAUTH_CLIENT_SECRET= GH_OAUTH_CLIENT_ID= goreleaser --skip-validate --skip-publish --rm-dist`
```

Comment:
was the need for this eliminated in a goreleaser update? it's always been required for me

Reply:
Originally they were needed, but no more as of c32bcee4bb4216ad3c749d314e93add6b5863c2d